includes/AMP/Sanitization.php

```diff
@@ -415,7 +461,7 @@ protected function get_sanitizers(): array {
 
 			$sanitizers = array_merge(
 				[
-					\AMP_Dev_Mode_Sanitizer::class => [
+					AMP_Dev_Mode_Sanitizer::class => [
```

Comment:
Why change this?

Reply:
Because this whole part was not actually working.

`AMP_Dev_Mode_Sanitizer` was missing from `scoper.inc.php` so this class never really existed in the plugin. It should exist under the `Google\Web_Stories_Dependencies` namespace, which it does now thanks to this change.

Referencing `\AMP_Dev_Mode_Sanitizer` (no namespace) was just plain wrong as that class doesn't exist when not using the AMP plugin.